taxi_service/settings.py

```diff
@@ -0,0 +1,77 @@
+import os
+from pathlib import Path
+
+
+SECRET_KEY = "django-insecure-012931238jjsoijef91-1-2ead9-0dfus0df9uijeifjsodif9"

```

Comment:
It's a security risk to have the `SECRET_KEY` hardcoded in the settings file, especially in a public repository. Consider using environment variables to manage sensitive information like the `SECRET_KEY`.